src/main/java/es/upm/etsisi/DefaultParameters.java

```diff
@@ -0,0 +1,94 @@
+package es.upm.etsisi;
+
+import es.upm.etsisi.models.Administrator;
+import es.upm.etsisi.models.User;
+
+import java.util.LinkedList;
+import java.util.Iterator;
+import java.util.Random;
+import java.text.Normalizer;
+
+public class DefaultParameters {
+    private static final int DEFAULT_PLAYERS = 6;
+    private static final int DEFAULT_TEAMS = 2;
+    private static final String[] DEFAULT_SURNAMES = { "Pérez", "Sánchez", "Jiménez", "Torres", "Blanco", "Navarro",
+            "Paredes", "Moreno" };
+    private static final String[] DEFAULT_NAMES = { "Luisa", "Manuel", "Kurt", "Sofia", "Robert", "Jose", "Ramón",
+            "Pablo" };
+    private static final String[] DEFAULT_TEAMNAMES = { "Alfa", "Beta", "Delta", "Omega", "Theta" };
+
+    public DefaultParameters() {
+    }
+
+    public static Administrator getDefaultAdmin() {
+        return new Administrator("admin@upm.es", "admin");
+    }
+
+    public static LinkedList<LinkedList<String>> getDefaultParticipants() {
+        LinkedList<LinkedList<String>> parameters = new LinkedList<>();
+        int j = 0;
+        for (int i = 0; i < DEFAULT_PLAYERS; i++) {
+            LinkedList<String> playerParams;
+            do {
+                playerParams = generatePlayer();
+            } while (parameters.contains(playerParams));
+            parameters.add(playerParams);
+            if (j < DEFAULT_TEAMS) {
+                LinkedList<String> teamParams;
+                do {
+                    teamParams = generateTeam(playerParams.get(1));
+                } while (!isValidTeam(parameters, teamParams));
+                parameters.add(teamParams);
+                j++;
+            }
+        }
+        return parameters;
+    }
```

Comment:
This method should be called as generateParticipants() and return a Participant list

Reply:
Create a new ParticipantList object inside and start adding. If the player is invalid it won't be added, and it will return an error. You can choose to do nothing with the returned error tho

Reply:
I guess you have to create an UserList too, if that's so, the generate Teams method have to get their players by arguments. So you can add them to both list

Reply:
Use a list as an argument maybe, think about it

Reply:
That's why I thought of generating parameters instead of objects. These parameters get passed to createPlayer and createTeam methods of Controller, which will generate both players and teams directly into the participantList and userList of the controller. That way we avoid having to copy a ParticipantList and UserList and then appending each element according by its type.

Will look into it though.